tests/models/t5/test_modeling_tf_t5.py

```diff
@@ -318,20 +318,6 @@ def test_generate_with_headmasking(self):
         # TODO: Fix head-masking according to PyTorch T5 model
         pass
 
-    @slow
-    def test_resize_embeddings(self):
```

Comment:
Because this test shared the name with the [boolean flag](https://github.com/huggingface/transformers/blob/c28d04e9e252a1a099944e325685f14d242ecdcd/tests/test_modeling_tf_common.py#L1151) that is used to determine whether to run the common resizing tests, it meant that T5 was not running the common resizing tests. This test is an incomplete version of the common tests, so removing it both removes duplicate test code and enables the run of the complete tests.